### DISCUSSION TITLE
Sending POST/GET when broadcast ends

### DISCUSSION THREAD

**Discussion by emresaracoglu** [streamlink/streamlink#4492] [Q&A] [answered]
Hello,

Is it possible to send a POST/GET when the broadcast/stream ends or when the recording/playing ends due to some other problem?

Thank you.

**Comment by gravyboat** [comment, marked as answer]
This is outside the scope of Streamlink. You would need to write a wrapper script.

**Reply by emresaracoglu** [reply]
Thank you for your answer, It is not possible for me to write this for now, as I do not know the Python language. I will request a feature.

**Reply by mkbloke** [reply]
It's likely to be closed if you do request it.  You don't need to know Python, you should be able to test the exit status of Streamlink with a shell script/Power Shell script/batch file.

You can find more appropriate forums to get help on ways to do that, but here is not the place.

**Reply by emresaracoglu** [reply]
@mkbloke _"exit status of Streamlink"_ How can I get the exit status? This is perhaps not the place, of course, but if you can give an example, I can do research on this subject.

**Reply by gravyboat** [reply]
@emresaracoglu You'll need to do an internet search on this and other topics. We cannot walk you through everything and if this continues I'm going to have to block you from posting as you're eating up a lot of our limited time as maintainers. If you are unable to perform these tasks you may wish to contract this work out to someone who is more familiar with these sorts of tasks.
